### PR TITLE
feat: extended the type of the request user message content with a list of content parts

### LINKED ISSUE #163: Support `request.message.content` as a list of content parts in chat/completions API

### PR DESCRIPTION
**Pull request by adubovik**
Resolves https://github.com/epam/ai-dial-sdk/issues/163

### CODE REVIEW

File: examples/echo/app.py [hunk 1]
```diff
@@ -15,17 +15,19 @@ async def chat_completion(
...
-            choice.append_content(last_user_message.content or "")
-
-            if last_user_message.custom_content is not None:
-                for attachment in (
-                    last_user_message.custom_content.attachments or []
-                ):
+            choice.append_content(
+                last_message.content
+                if isinstance(last_message.content, str)
+                else ""
```
Comment (vladisavvv) [inline examples/echo/app.py]: could we process the list case like streaming of the parts?

**Review by vladisavvv — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ class EchoApplication(ChatCompletion):
         # Generate response with a single choice
         with response.create_single_choice() as choice:
             # Fill the content of the response with the last user's content
-            choice.append_content(last_user_message.content or "")
+            choice.append_content(last_user_message.text)
```
Comment (vladisavvv) [inline README.md]: `text()`

File: examples/langchain_rag/app.py [hunk 3]
```diff
@@ -61,7 +61,7 @@ async def chat_completion(
 
         with response.create_single_choice() as choice:
             message = request.messages[-1]
-            user_query = message.content or ""
+            user_query = message.text
```
Comment (vladisavvv) [inline examples/langchain_rag/app.py]: `text()`

File: README.md (hunk 2 above)
Comment (adubovik) [reply README.md]: fixed

File: examples/langchain_rag/app.py (hunk 3 above)
Comment (adubovik) [reply examples/langchain_rag/app.py]: fixed

**Review by vladisavvv — APPROVED** [review]

**Pull request merged**